common/components/time/time.yaml

```diff
@@ -11,6 +11,18 @@ params:
     type: string
     required: false
     description: Classes to add to the element.
+  - name: displayAsTag
+    type: boolean
+    required: false
+    description: Whether to display the time element within a GOV.UK Tag component
+  - name: relative
+    type: boolean
+    required: false
+    description: Whether to show a relative time
+  - name: imminentOffset
+    type: number
+    required: false
+    description: The offset (in minutes) at which to use a "yellow" tag instead of green. Inside this offset the tag will be yellow, outside of it green, and if the time has past, it will be red.
```

Comment:
Could we get some examples in the Component library of green / yellow / red styling?

Reply:
Unfortunately not all states...I can show red and green. But yellow is based on an offset and we can't set a dynamic date within the yaml file that will always fall within that offset.

Although the only way might be to set a really high number and a date in the future that we _might_ not reach.

Reply:
Happy with red and green for now, the colour itself is not a super high risk feature.